python/nav/bitvector.py

```diff
@@ -31,7 +33,7 @@ class BitVector(object):
     def __init__(self, octetstring):
         self.vector = array.array("B", octetstring)
 
-    def __str__(self):
+    def to_bytes(self):
```

Comment:
Just a question: How does this affect printing/logging of BitVector objects (is this an accepted API in Python3?)